.github/workflows/docker-openmetadata-server.yml

```diff
@@ -83,6 +86,18 @@ jobs:
           asset_name: openmetadata-${{ inputs.DOCKER_RELEASE_TAG }}.tar.gz
           asset_content_type: application/tar.gz
 
+      - name: Docker files upload
+        uses: softprops/action-gh-release@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Hello @safayavatsal - Is this required when using `softprops/action-gh-release@v2`?